btr-web/btr-main-app/components/individual-person/AddNew.vue

```diff
@@ -337,7 +339,7 @@ function validateForm () {
     percentOfShares: significantIndividual.value.percentOfShares,
     percentOfVotes: significantIndividual.value.percentOfVotes,
     controlOfShares: significantIndividual.value.controlType.sharesVotes,
-    otherReasons: significantIndividual.value.controlType.other,
+    otherReason: significantIndividual.value.controlType.other,
```

Comment:
hey patrick, we've got this as 'otherReasons' in more places than 'otherReason'. Can you update it the other way so its named consistently throughout the code? When I looked I just saw 2 other places to update it